assets/src/js/_acf-validation.js

```diff
@@ -1177,9 +1177,10 @@
 										new CustomEvent(
 											'acf/block/has-error',
 											{
-												acfBlocksWithValidationErrors: [
-													block,
-												],
+												detail: {
+													acfBlocksWithValidationErrors:
```

Comment:
See [my comment above/below](https://github.com/WordPress/secure-custom-fields/pull/281#discussion_r2651274916) about the name mismatch.

Reply:
Done!